nodes/core/core/80-template.html

```diff
@@ -80,6 +80,11 @@ <h3>Details</h3>
     <p>It is possible to use a property from the flow context or global context. Just use <code>{{flow.name}}</code> or <code>{{global.name}}</code>.
     <p><b>Note: </b>By default, <i>mustache</i> will escape any HTML entities in the values it substitutes.
        To prevent this, use <code>{{{triple}}}</code> braces.
+    <h4>Migrating Template Libraries</h4>
+    <p>Before Node-RED 0.18.4, library template files were saved in the <i>functions</i> library located in the
+    Node-RED user directory (typically <code>.node-red</code> in the user's home directory).  To access these saved templates, 
+    after upgrading, make a copy of the template files and directories in <code>.node-red/functions</code> to
+    <code>.node-red/templates</code>.</p>
 </script>
```

Comment:
Can we remove this text? My assumption is this will affect a pretty small set of users so would rather not tarnish the node info with this. We can make sure it is highlighted in the release blog post.